File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    res=''

    if number %3 * number %5 * number %7 !=0: # number has no factors in 3/5/7
        return str(number)

    if number%3==0: 
        res+='Pling'
    if number%5==0:
        res+='Plang'
    if number%7==0:
        res+='Plong'        

    return res

File: solutions/dart/raindrops/1/lib/raindrops.dart
class Raindrops {
  String convert(num a) {
    String result = "";
    if (a % 3 == 0) {
      result += "Pling";
    }
    if (a % 5 == 0) {
      result += "Plang";
    }
    if (a % 7 == 0) {
      result += "Plong";
    }
    if (result == "") {
      result = a.toString();
    }
    return result;
  }
   
}
